### PR TITLE
Publish release

### PR DESCRIPTION
**Pull request by Droogans**

### CODE REVIEW

File: src/snappit.ts [hunk 1]
```diff
@@ -112,7 +112,10 @@ export class Snappit {
         // No baseline image
         } else {
             newShot.saveToPath(filePath);
-            throw new ScreenshotNotPresentException();
+
+            if (this.config.throwNoBaseline) {
+                throw new ScreenshotNotPresentException();
+            }
```
Comment (Droogans) [inline src/snappit.ts]: 🙌

**Comment by Droogans** [comment]
@jj4th "No previous screenshot found" for

```
      1) should throw an error and save if the screenshot is a different size
      2) should throw an error and save if the screenshot is different above threshold
      3) should not throw an error or save if the screenshot is different below threshold
      4) should not throw an error or save if the screenshot shows no difference
      5) should take a fullscreen screenshot that shows no difference
```

I think those are intentional, so I'll leave it for now.

**Comment by jj4th** [comment]
Yeah I need to rework the tests a bit to make it work on different systems.

**Comment by Droogans** [comment]
This branch is officially a pile of mess.

It's passing, but it's still probably not what we're looking for long term.

Merge and iterate?

**Comment by jj4th** [comment]
Hmm, no I want to fix those tests first. @Droogans

**Comment by jj4th** [comment]
travis is being silly

**Comment by Droogans** [comment]
Retriggered it since it hang for ~40 minutes.

File: test/snappit.spec.ts [hunk 2]
```diff
@@ -266,6 +266,12 @@ describe("Snappit", () => {
             expect(fs.existsSync(path)).to.eql(true);
         });
 
+        it("should handle an oversized element that is larger than the viewport size", async () => {
+            $("#toggle-tall").click();
+            await snap("chrome-throw-no-oversized-crop.png", $("#tall-div")).catch((err) => err);
+            await setViewportSize(driver, {width: 100, height: 100});
+            await snap("chrome-throw-no-oversized-crop.png", $("#tall-div"));
+        });
```
Comment (Droogans) [inline test/snappit.spec.ts]: If you're going to resize it that small, you could ditch the tall div altogether.

You could also drop all tall div stuff in tests.

Comment (jj4th) [reply test/snappit.spec.ts]: Hmm... ok if you think that's a good direction, making that change.

**Pull request merged**